src/lib/viewers/controls/_styles.scss

```diff
@@ -2,8 +2,8 @@
 @import '~box-ui-elements/es/styles/deprecated';
 
 $bp-controls-background: fade-out($black, .2);
-$bp-controls-opacity: .8;
-$bp-controls-outline: inset 0 0 0 1px fade-out($white, .5), 0 1px 2px fade-out($black, .9);
+$bp-controls-opacity: .9;
+$bp-controls-outline: inset 0 0 0 1px fade-out($bdl-gray-20, 0), 0 1px 2px fade-out($black, .9);
```

Comment:
We should be able to remove the `fade-out` directive here, since we're passing a zero value to it.

Reply:
Thanks Jared. Shouldn't be zero. Updated!